src/main/osd/osd_elements.c

```diff
@@ -835,7 +832,7 @@ static void osdElementGpsSats(osdElementParms_t *element)
 
 static void osdElementGpsSpeed(osdElementParms_t *element)
 {
-    tfp_sprintf(element->buff, "%3d%c", osdGetSpeedToSelectedUnit(gpsSol.groundSpeed), osdGetSpeedToSelectedUnitSymbol());
+    tfp_sprintf(element->buff, "%c%d%c", SYM_SPEED, osdGetSpeedToSelectedUnit(gpsSol.groundSpeed), osdGetSpeedToSelectedUnitSymbol());
```

Comment:
Is it intentional to change the format for the speed here?

Reply:
Yes. Adding the prefix symbol makes the things strange if we add spaces, like we did with the altitude. I think is better to remove them. But I'm open to other opinions.

Reply:
There's two sides to that - keeping numeric values at a fixed length is desirable where possible, as it will keep the display from jumping around when the value is around a decimal jump, making it easier to read. But I agree that having a variable number of spaces between symbol and number does not look great. Maybe we should create a function that allows us to print a number into a fixed number of characters, and prepend the symbol to this - this could probably be used for good readability and look in a number of places.

Reply:
To me, having the prefix char jumping will be as bad as having the decimal jumping.

In this case, speed, we usually will have two digits, so we can have a fixed length that will have usually only space or none between the prefix and the number, but we have other elements like altitude where this does not happen.

I have looked at iNav, they use the OSD a lot more than ours, to see what they do, but for this element, the speed, they don't have a prefix symbol. For other elements they align them to the right, adding spaces, but they have a more complex function that if the value is too big they change the scale (from meters to kilometers for example) to have them in usually 3 chars long.

For this element, that will have a maximum of 3 chars long and usually used one, another solution can be to add zeros and not spaces but to me produces an OSD less clean.